packages/angular_devkit/core/src/jobs/README.md

```diff
@@ -0,0 +1,207 @@
+# Jobs
+
+Jobs is the Angular DevKit subsystem for scheduling and running generic functions with clearly 
+typed inputs and outputs. A Job is 
```

Comment:
"A Job is" is not a sentence, would ❤️ to see a longer definition here.

Reply:
Clarifying this. I'm reworking the README a bit and by EOD I'll have a proper description to replace that non-sentence.